src/TLuaInterpreter.cpp

```diff
@@ -2462,9 +2472,15 @@ int TLuaInterpreter::openUserWindow( lua_State *L )
     {
         luaSendText = lua_tostring( L, 1 );
     }
+
+    bool loadLayout = true;
+    if( ! lua_isnoneornil(L, 2) && lua_isboolean(L, 2) ) {
+        loadLayout = lua_toboolean(L, 2);
+    }
+
     Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
     QString text(luaSendText.c_str());
```

Comment:
As long as I know what code I should be using I'll make a point to update the code I'm nearest to while working on it.  I'm not to sure what to do about this stuff though and I would like to learn so I can use that when I start making the IRC related functions and changes.